cea/interfaces/dashboard/inputs/routes.py

```diff
@@ -106,6 +106,25 @@ def route_geojson(db):
     return jsonify(json.loads(table_df.to_json(show_bbox=True)))
 
 
+@blueprint.route('/geojson/networks/<type>')
+def route_geojson_networks(type):
+    locator = cea.inputlocator.InputLocator(current_app.cea_config.scenario)
+    # TODO: Get a list of names and send all in the json
+    name = ''
+    edges = locator.get_network_layout_edges_shapefile(type, name)
+    nodes = locator.get_network_layout_nodes_shapefile(type, name)
+    network_json = df_to_json(edges)
+    network_json['features'].extend(df_to_json(nodes)['features'])
+    return jsonify(network_json)
+
+
+@blueprint.route('/geojson/others/streets')
```

Comment:
why is the route `/geojson/others/streets` and not `/geojson/streets`?

Reply:
`route_geojson` will catch `/geojson/streets` and throw a `404` because it's not in `inputs.yaml`. I could include it into `inputs.yaml` file or create a condition in `route_geojson` .